app/policies/lti_policy.rb

```diff
@@ -0,0 +1,10 @@
+# Policy for LTI controller
+class LtiPolicy < ApplicationPolicy
+  skip_pre_check :role_exists?
+
+  default_rule :manage?
+
+  def manage?
+    Role.where(user: real_user, type: 'Instructor').present?
```

Comment:
change so that users need to be logged in for `choose_course` GET and POST, but for GET any user can access, and for POST only the instructor of that course can assign the course to the LTI deployment